blocks/show-hide-group/block.json

```diff
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://json.schemastore.org/block.json",
+	"apiVersion": 2,
+	"name": "happyprime/show-hide-group",
+	"title": "Show Hide Group",
+	"textdomain": "happy-prime",
+	"category": "widgets",
+	"description": "Creates accessible summaries that can be toggled to show or hide additional details.",
+	"icon": "hidden",
+	"attributes": {
+		"allIds": {
+			"type": "string"
+		},
+		"blockCount": {
+			"type": "number"
+		}
+	},
+	"supports": {
+		"html": false,
+		"multiple": false
```

Comment:
I think we could allow multiple support without too much trouble - I'll leave a couple comments throughout for how I think it could be done.